fig/falcon/models.py

```diff
@@ -5,10 +5,16 @@
 
 
 class Event(dict):
-    def __init__(self, event_string):
+    def __init__(self, event_string, feed_id):
         event = json.loads(event_string.decode('utf-8'))
+        self.feed_id = feed_id
         super().__init__(event)
 
+    def __eq__(self, other):
+        if not isinstance(other, Event):
+            return False
+        return dict.__eq__(self, other) and self.feed_id == other.feed_id
```

Comment:
@isimluk what do you think?

Reply:
I would use `super()` instead of `dict__eq__` if you would ask me.

Reply:
was going off https://lgtm.com/rules/9990086/

Reply:
:man_shrugging:

Reply:
Changed it to super() 👍🏼

Reply:
Super!